packages/flame_svg/test/svg_test.dart

```diff
@@ -93,5 +93,39 @@ void main() {
         );
       },
     );
+
+    testWidgets(
+      'render sharply with viewfinder zoom',
```

Comment:
It would be great if we could use the "new" way for doing the golden test:

https://github.com/flame-engine/flame/blob/main/packages%2Fflame%2Ftest%2Fcamera%2Fcamera_component_test.dart#L16

Reply:
The `testGolden` method seems to be defined in the `flame_test` package. With a search locally, I've found that only files under `packages/flame/test/` or `packages/flame_test/` references this method. Should this test be moved into `packages/flame/test`? or should the `flame_test` package be added as a dev dependency?

Reply:
Applied in 85e885b2c81f809cd9aa64c25d4654289c5bd656. The test now uses the provided game instance with `camera.viewfinder.zoom` instead of a `Transform` component